pkg/collector/corechecks/oracle-dbm/oracle.go

```diff
@@ -283,6 +295,16 @@ func (c *Check) Configure(integrationConfigDigest uint64, rawInstance integratio
 	c.tags = append(c.tags, fmt.Sprintf("dbms:%s", common.IntegrationName), fmt.Sprintf("ddagentversion:%s", c.agentVersion))
 
 	c.tagsString = strings.Join(c.tags, ",")
+
+	c.fqtCache = ttlcache.NewCache()
```

Comment:
I'm not sure what the proper name would be for those cache instances but reading `fqtCache`, my intuition would be that we're caching the fqt (or that we're caching plans for the `planCache`) and not the fact that it was already sent during the cache's ttl.

Reply:
Renamed to `fqtEmitted`.